test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -0,0 +1,418 @@
+      return {
+        appID = c.getHMIAppId(pAppId),
+        moduleType = pModuleType
+      }
+    end,
+    hmiResponseParams = function(pAllowed)
+      return {
+        allowed = pAllowed
+      }
+    end,
+  },
+  OnInteriorVehicleData = {
+    appEventName = "OnInteriorVehicleData",
+    hmiEventName = "RC.OnInteriorVehicleData",
+    hmiResponseParams = function(pModuleType)
+      return {
+        moduleData = c.getAnotherModuleControlData(pModuleType)
+      }
+    end,
+    responseParams = function(pModuleType)
+      return {
+        moduleData = c.getAnotherModuleControlData(pModuleType)
+      }
+    end
+  },
+  OnRemoteControlSettings = {
+    hmiEventName = "RC.OnRemoteControlSettings",
+    hmiResponseParams = function(pAllowed, pAccessMode)
+      return {
+        allowed = pAllowed,
+        accessMode = pAccessMode
+      }
+    end
+  }
+}
+
+function c.getAppEventName(pRPC)
+  return rcRPCs[pRPC].appEventName
+end
+
+function c.getHMIEventName(pRPC)
+  return rcRPCs[pRPC].hmiEventName
+end
+
+function c.getAppRequestParams(pRPC, ...)
+  return rcRPCs[pRPC].requestParams(...)
+end
+
+function c.getAppResponseParams(pRPC, ...)
+  return rcRPCs[pRPC].responseParams(...)
+end
+
+function c.getHMIRequestParams(pRPC, ...)
+  return rcRPCs[pRPC].hmiRequestParams(...)
+end
+
+function c.getHMIResponseParams(pRPC, ...)
+  return rcRPCs[pRPC].hmiResponseParams(...)
+end
+
+function c.getMobileSession(pAppId)
+  if not pAppId then pAppId = 1 end
+  return test["mobileSession" .. pAppId]
+end
+
+function c.getHMIconnection()
+  return test.hmiConnection
+end
+
+function c.subscribeToModule(pModuleType, pAppId)
+  if not pAppId then pAppId = 1 end
+  local rpc = "GetInteriorVehicleData"
+  local subscribe = true
+  local mobSession = c.getMobileSession(pAppId)
+  local cid = mobSession:SendRPC(c.getAppEventName(rpc), c.getAppRequestParams(rpc, pModuleType, subscribe))
+  EXPECT_HMICALL(c.getHMIEventName(rpc), c.getHMIRequestParams(rpc, pModuleType, pAppId, subscribe))
+  :Do(function(_, data)
+      test.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", c.getHMIResponseParams(rpc, pModuleType, subscribe))
+    end)
+  mobSession:ExpectResponse(cid, c.getAppResponseParams(rpc, true, "SUCCESS", pModuleType, subscribe))
+end
+
+function c.unSubscribeToModule(pModuleType, pAppId)
+  local rpc = "GetInteriorVehicleData"
+  local subscribe = false
+  local mobSession = c.getMobileSession(pAppId)
+  local cid = mobSession:SendRPC(c.getAppEventName(rpc), c.getAppRequestParams(rpc, pModuleType, subscribe))
+  EXPECT_HMICALL(c.getHMIEventName(rpc), c.getHMIRequestParams(rpc, pModuleType, pAppId, subscribe))
+  :Do(function(_, data)
+      test.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", c.getHMIResponseParams(rpc, pModuleType, subscribe))
+    end)
+  mobSession:ExpectResponse(cid, c.getAppResponseParams(rpc, true, "SUCCESS", pModuleType, subscribe))
+end
+
+function c.activate_app(pAppId)
```

Comment:
@GetmanetsIrina Please use camelCase for the function names

Reply:
@dboltovskyi updated in 680ea88fd7e466544d3a9249e197deaac46aa429 commit.